client/src/components/Auth/SignUp.jsx

```diff
@@ -8,20 +8,18 @@ import styled from "styled-components";
 
 import axios from "axios";
 
-
-import Modal, { ModalProvider, BaseModalBackground } from "styled-react-modal";
-
-
-const StyledModal = Modal.styled`
-  width: 20rem;
-  height: 20rem;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  background-color: white;
-  opacity: ${props => props.opacity};
-  transition: opacity ease 500ms;
-`;
+import Modal from "styled-react-modal";
+
+// const StyledModal = Modal.styled`
+//   width: 20rem;
+//   height: 20rem;
+//   display: flex;
+//   align-items: center;
+//   justify-content: center;
+//   background-color: white;
+//   opacity: ${props => props.opacity};
+//   transition: opacity ease 500ms;
+// `;
```

Comment:
is this code still needed?

Reply:
Hesitant to delete since I didn't make it or comment it out. For now, I'll keep it in assuming someone else on the team plans on using it for later.

Reply:
Went ahead and removed this for now. If we need to refer to it later, we can look at the prior commit.